Userland/Libraries/LibSQL/TreeNode.cpp

```diff
@@ -94,15 +97,21 @@ bool TreeNode::insert(Key const& key, TreeNode *right)
         }
     }
 
+#ifdef BTREE_DEBUG
     dbgln("Inserting in this node");
+#endif
```

Comment:
there is something called `dbgln_if`
it essentially does the same, but looks neater and is the preferred way of doing this